File: condicionales/03.py
t1 = float(input('¿Cuanto fue su note en la PRACTICA 01: '))
t2 = float(input('¿Cuanto fue su note en la PRACTICA 02: '))
t3 = float(input('¿Cuanto fue su note en la PRACTICA 03: '))
t4 = float(input('¿Cuanto fue su note en la PRACTICA 04: '))
emc = float(input('¿Cuanto fue su note en el medio curso: '))
efc = float(input('¿Cuanto fue su note en el fin de curso: '))
promedio = (((t1+t2+t3+t4)/4)+emc+efc)/3
if promedio >= 10.5:
    print(f'Usted esta aprobado con {promedio}')
elif promedio < 10.5:
    print(f'Su promedio es {promedio}')
    rpta = int(input('¿Desea ir al sustitutorio?:\n[1]SI\n[2]NO\n'))
    if rpta == 1:
        susti = float(input('¿Cual fue su nota?: '))
        if emc < efc and susti > emc:
            promedio = (((t1+t2+t3+t4)/4)+susti+efc)/3
            print(f'Su nuevo promedio es {promedio}')
        elif emc > efc and susti < emc:
            promedio = (((t1+t2+t3+t4)/4)+emc+susti)/3
            print(f'Su nuevo promedio es {promedio}')
        else:
            print('Su promedio se mantiene porque es menor a las obtenidas')
else:
    print('Tendra que llevar el curso nuevamente')

File: condicionales/03.cpp
// Dado las 4 notas de tareas académicas, el emc y el de fin de curso,
// calcule su promedio e imprima el mensaje respectivo o si ingresa al sustitutorio.
// En caso afirmativo, reemplazar la menor nota (emc o efc) con el sustitutorio y hallar
// el nuevo promedio final.

#include "iostream"
using namespace std;

int main()
{
    float t1,t2,t3,t4,emc,efc,susti,promedio;
    int rpta;
    cout << "¿Cual fue su nota en la PRACTICA 01?: ";
    cin >> t1;
    cout << "¿Cual fue su nota en la PRACTICA 02?: ";
    cin >> t2;
    cout << "¿Cual fue su nota en la PRACTICA 03?: ";
    cin >> t3;
    cout << "¿Cual fue su nota en la PRACTICA 04?: ";
    cin >> t4;
    cout << "¿Cual fue su nota en el medio curso?: ";
    cin >> emc;
    cout << "¿Cual fue su nota en el fin curso?: ";
    cin >> efc;
    promedio = (((t1+t2+t3+t4)/4)+emc+efc)/3;
    if (promedio >= 10.5)
    {
        cout << "Usted esta aprobado con " << promedio;
    }
    else if (promedio < 10.5)
    {
        cout << "Su promedio es: " << promedio;
        cout << "¿Desea ir al sustitorio?\n[1]SI\n[2]NO\n";
        cin >> rpta;
        if (rpta==1)
        {
            cout << "¿Cual fue su nota?: ";
            cin >> susti;
            if (emc < efc and susti > emc)
            {
                promedio = (((t1+t2+t3+t4)/4)+susti+efc)/3;
                cout << "Su nuevo promedio es: " << promedio;
            }
            else if (emc > efc and susti < emc)
            {
                promedio = (((t1+t2+t3+t4)/4)+emc+susti)/3;
                cout << "Su nuevo promedio es: " << promedio;
            }
            else
            {
                cout << "Su promedio se mantiene porque es menor a las obtenidas";
            }
        }
        else
        {
            cout << "Tendra que llevar el curso nuevamente";
        }
    }
    return 0;
}